wire-ios/Wire-iOS Tests/ConversationDetail/GroupDetailsViewControllerSnapshotTests.swift

```diff
@@ -42,14 +42,16 @@ final class GroupDetailsViewControllerSnapshotTests: ZMSnapshotTestCase {
         otherUser.isConnected = true
         otherUser.handle = "bruno"
         otherUser.accentColorValue = .brightOrange
+
+        userSession = UserSessionMock()
```

Comment:
There's a mock self user in this test, shouldn't we pass that in to the user session?

Reply:
Done!